src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package org.example;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String colorRandom = ColorSupplier.getRandomColor().toString(); //1
+        Random random = new Random();
+        int numberRandom = random.nextInt(100) + 1; //2
```

Comment:
100 is a magic number, avoid using magic numbers

should be constant with proper name
```suggestion
        int numberRandom = random.nextInt(100) + 1; //2
```